app/components/provider_interface/organisation_permissions_review_card_component.rb

```diff
@@ -17,27 +17,20 @@ def permission_rows
       ProviderRelationshipPermissions::PERMISSIONS.map do |permission_name|
         {
           key: label_for(permission_name),
-          value: providers_to_render_for(permission_name),
+          value: providers_with_permission(permission_name),
```

Comment:
This has now changed the layout to use `p` tags instead of a `ul`.

The prototype uses both of these formats (`p` in the onboarding flow, and `ul` in the org settings pages), and this is less effort codewise now we have the `paragraph_format` option